src/types/index.ts

```diff
@@ -286,4 +286,6 @@ export interface TransactionInfo {
   transactionId: string
 }
 
-export type TaskState = '' | 'open' | 'closed'
+export type MyTaskState = '' | 'open' | 'closed'
+
+export type TaskState = '' | 'assigned' | 'unassigned'
```

Comment:
I don't think it's a reasonable way to handle this. State should still be open and closed.
AssignState can be assigned and unassigned.